dns/octodns-docker/Dockerfile

```diff
@@ -12,9 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# python:2-alpine as of July 11th, 2019
-ARG BASE_IMAGE=python@sha256:9a2db929c9e4c65118cb1da9ddb1d64f3ba8e4a722f07f3f2d34b58b1e8556b1
-FROM ${BASE_IMAGE}
```

Comment:
Why was this abstraction removed?

Reply:
No reason in particular, I think I removed it when I was trying out the image digests.

Reply:
It's helpful to have this there, because you can then test by passing in a --build_arg for BASE IMAGE

Reply:
OK, I've added the abstraction back.